src/app/components/IRLCollectibles/IRLCollectibles.tsx

```diff
@@ -0,0 +1,58 @@
+import { Grid, Typography } from '@material-ui/core';
+import ChrisMazdzer from 'app/assets/winterGrid/ChrisMazdzer.jpg';
+import GusKenworthy from 'app/assets/winterGrid/GusKenworthy.jpg';
+import JohnShuster from 'app/assets/winterGrid/JohnShuster.jpg';
+import KendallCoyne from 'app/assets/winterGrid/KendallCoyne.jpg';
+import TinaMaze from 'app/assets/winterGrid/TinaMaze.jpg';
+import { dolphinServiceLinks } from 'app/constants/constants';
+import useTranslation from 'app/hooks/useTranslation';
+import styles from './IRLCollectibles.module.scss';
+
+export const IRLCollectibles = () => {
+  const t = useTranslation();
```

Comment:
Are you sure this is a descriptive name? Don't you think it would be better to have a name that matches the content of this variable?

Reply:
It's not as descriptive, but it's a standard in the project that `{t(' ')}` means translation. I saw it on other projects as well. I think it's from another i18n library that take that same variable as translation.
https://react.i18next.com/latest/usetranslation-hook --> It's not the same library that we're using, but I think that from here came the standard.

Reply:
I didn't know this! Good to know, thank you for the explanation.